README_new.md

```diff
@@ -0,0 +1,617 @@
+# Orion Context Broker
+
+This is the code repository for the Orion Context Broker, the reference implementation of the Publish/Subscribe Context Broker GE.
+
+This project is part of FIWARE.
+
+* [FIWARE Site](http://www.fiware.org)
+* [FIWARE Catalogue for Orion](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker)
+
+Any feedback on this documentation is highly welcome, including bugs, typos
+or things you think should be included but aren't. You can use [github issues](https://github.com/telefonicaid/fiware-orion/issues/new) to provide feedback.
+
+For documentation previous to Orion 0.23.0 please check the old manuals at FIWARE public wiki:
+
+* [Orion Context Broker - Installation and Administration Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_Installation_and_Administration_Guide)
+* [Orion Context Broker - User and Programmers Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_User_and_Programmers_Guide)
+
+## GEi overall description
+
+Orion is a C++ implementation of the NGSI9/10 REST API binding developed as a part of the FIWARE platform.
+
+The Orion Context Broker is an NGSI9/10 server implementation to manage context information and context information availability. Using the Orion Context Broker, you are able to register context elements and manage them through updates and queries. In addition, you can subscribe to context information so when some condition occurs (e.g. an interval of time has passed or the context elements have changed) you receive a notification. These usage scenarios and the Orion Context Broker features are described in this document.
+
+If this is your first contact with Orion Context Broker, it is highly recommended to have a look to the brief [Quick Start guide](doc/md/quick_start_guide.md).
```

Comment:
Fixed in de740476a6006285a458cff382e091e1c9c191f5 and 217e8111c2e2b1ca24380ba30e4dfd75f3109301